pkg/specgen/volumes.go

```diff
@@ -112,7 +112,7 @@ func GenVolumeMounts(volumeFlag []string) (map[string]spec.Mount, map[string]*Na
 			}
 		}
 
-		if strings.HasPrefix(src, "/") || strings.HasPrefix(src, ".") || isHostWinPath(src) {
```

Comment:
Although not broken, this check was not previously working as the author seemed to intend since it was only effective for `'.'` exactly or for paths starting with `'..'`. All other relative paths were resolved to absolute before getting to this point meaning that (with the previous exceptions) they only needed to check for the `'/'` prefix.